### ISSUE TITLE
youtube tab

### ISSUE THREAD

**Issue by Gibbz** [openWMail/openWMail#16]
* Which version of openWMail are you using? [e.g. 1.3.1]
latest?

* Which Operating System are you using? [e.g. Windows 10, Mac OSX 10, Ubuntu 14.04]
manjaro

Would love to see a youtube tab for uploading/managing videos!

**Comment by Gibbz** [comment]
ps thanks for forking! :)

**Comment by geekgonecrazy** [comment]
@Gibbz Thanks for using!  :+1: How integrated would you see this being?  Along the levels of Google Keep?  No real notifications just kind of conveniently available?

**Comment by Gibbz** [comment]
Yep, just need to upload a weekly video. Dont care for anything else apart
from that ☺️

**Comment by Gibbz** [comment]
Has there been any progress on this?
Really loving this app btw 👍
